test/test_basic.py

```diff
@@ -0,0 +1,10 @@
+from journalpump import journalpump
+from tempfile import NamedTemporaryFile
+import json
+
+
+def test_basic():
```

Comment:
I think all this and more are covered by the unittests in the file test_journalpump.py